packages/tools/nodeEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -559,7 +559,7 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         />
                     </LineContainerComponent>
                     <LineContainerComponent title="FILE">
-                        <FileButtonLineComponent label="Load" onClick={(file) => this.load(file)} accept=".json" />
+                        <FileButtonLine label="Load" onClick={(file) => this.load(file)} accept=".json" />

```

Comment:
I'd like to see us standardize the naming as we clean up the duplication, but that might be best as its own "rename only" PR.  Specifically, I think we should either have Component as a suffix in all of our React components, or in none of them (unless there's a distinction I'm not seeing between the two).

Reply:
Agree that the naming should be standardized everywhere.